<launch>
  <!-- <node name="tf_bc" -->
  <!--       pkg="recog_pkg" type="tf_bc.py" output="screen" /> -->
  
  <node name="tf_broadcaster"
        pkg="recog_pkg" type="tf_broadcaster.py"
        output="screen">
    <param name="button" type="string" value="button1"/>
    <remap from="~input" to="/camera_3d_point" />
  </node>

  <node name="two2three"
        pkg="recog_pkg" type="two2three.py"
        output="screen">
    <remap from="~output" to="/camera_3d_point"/>
  </node>

  <node name="find_button_hand"
        pkg="recog_pkg" type="find_button_hand.py" output="screen">
    <remap from="~output" to="/hand_camera_2d_point" />
  </node>

  <!-- <node name="two2three_hand" -->
  <!--       pkg="recog_pkg" type="two2three_hand.py" -->
  <!--       output="screen" /> -->
  
</launch>
